practice-app/website/api/doc/event_GET.yml

```diff
@@ -84,6 +84,10 @@ responses:
           type: string
           description: The weather icon id, taken using OpenWeatherMap API.
           example: 03d
-  
+  400:
+    description: "Event ID is not correct"
+
+  404:
```

Comment:
we return 400 for this case but I am not sure which one is correct

Reply:
For which case? Do you mean line 87 or line 90?

Reply:
I think since endpoint is correct we should return 400. I don't see any difference to Event Id not correct

Reply:
I agree with Elif Sema, for example we return 400 if there is an attempt to get a badge point for a non-existent badge. I think 404 is more related to an endpoint path etc.

Reply:
In [this](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status) document, it states that 404 response can not only be used when the URL is not recognized(which is the case you stated above), but also when the endpoint is valid but "the resource itself does not exist". I think that the latter is the case here, since the event does not exist. Hence, 404 response could be suitable in our case. Maybe we should discuss it as whole team and perhaps ask it to our teaching assistant.

Reply:
I converted 404 to 400 in order to preserve consistency in response status codes in the projects.